models/schema.yml

```diff
@@ -8,7 +8,7 @@ sources:
       - name: events
   - name: derived
     schema:  "{{target.schema ~ '_derived'}}"
-    database: "{{ var('snowplow__database', target.database) if target.type not in ['databricks', 'spark'] else var('snowplow__databricks_catalog', 'hive_metastore') if target.type in ['databricks'] else var('snowplow__atomic_schema', 'atomic') }}"
+    # database: "{{ var('snowplow__database', target.database) if target.type not in ['databricks', 'spark'] else var('snowplow__databricks_catalog', 'hive_metastore') if target.type in ['databricks'] else var('snowplow__atomic_schema', 'atomic') }}"
```

Comment:
why is this commented out?

Reply:
because spark doesnt have a database and there should not be an attribute ( for the rest is not like it was needed ) do you think its an issue?

Reply:
in the media player packages it is solved as none. I'm sure there was a reason why it was added for the rest, perhaps something to do with integration tests, better not change it even if it doesn't fail currently.